### DISCUSSION TITLE
Gemini with a little humor

### DISCUSSION THREAD

**Discussion by nwish** [blakeblackshear/frigate#15280] [General]
I added “with a little humor” into the default GenAI prompt and it did not disappoint 🤣😅

![IMG_4832](https://github.com/user-attachments/assets/7b941ee5-a065-4ba0-bca4-b43bdf5e04b0)
![IMG_4833](https://github.com/user-attachments/assets/b4e5674f-d323-4d50-a722-ae305adafb39)
![IMG_4834](https://github.com/user-attachments/assets/4ec9e33f-cb69-4566-8f20-df5d05d1ebc3)
![IMG_4835](https://github.com/user-attachments/assets/cfaabc0c-a1ab-4488-bc9d-cabf496c6b14)
![IMG_4836](https://github.com/user-attachments/assets/684e4dde-8554-4d90-80a6-84e1c4a6b0a3)
![IMG_4837](https://github.com/user-attachments/assets/ccdf2e04-3f30-4f4e-bd5f-5e98686f7053)

**Comment by psCLE** [comment]
Thanks for the idea, I was looking for Gemini examples. A thread idea would be users' AI examples. As of now I am using the stock requests in the documentation but looking for something more useful.
